packages/scripts/vue.config.js

```diff
@@ -18,8 +18,6 @@ module.exports = {
     config.externals = {
       'bootstrap': 'bootstrap'
     }
-    config.output.filename = 'js/scripts/[name].js',
```

Comment:
lines can be removed , were needed to have /scripts/ subpath

Reply:
I think otherwise we'll get a chunk hash in the name

Reply:
there is a separate , defidated flag for that